src/System.Device.Gpio/System/Device/Gpio/GpioController.Linux.cs

```diff
@@ -42,10 +42,13 @@ private static GpioDriver GetBestDriverForBoard()
                         {
                             return new RaspberryPi3Driver();
                         }
+                        /* Commenting out as HummingBoard driver is not implemented yet, will be added back after implementation 
```

Comment:
I don't think that we have a coding guideline for comments, but can we change this to instead be:
```c#
                        // Commenting out as HummingBoard driver is not implemented yet, will be added back after implementation 
                        // https://github.com/dotnet/iot/issues/76
                        //if (match.Groups[1].Value == HummingBoardHardware)
                        //{
                        //    return new HummingBoardDriver();
                        //} 
```

I think it looks cleaner and more aligned.

Reply:
Sure, will update